README.md

```diff
@@ -144,7 +144,7 @@ ios/tmp.xcconfig
 6. create new build phase for the scheme which will generate "tmp.xcconfig" before each build exposing values to Build Settings and Info.plist (this snippet has to be placed after "echo ... > tmp/envfile" if [approach explained below](#ios-multi-scheme) is used)
 
 ```
-"${SRCROOT}/../node_modules/react-native-config/ios/ReactNativeConfig/BuildXCConfig.rb" "${SRCROOT}/.." "${SRCROOT}/tmp.xcconfig"
+"${SRCROOT}/../node_modules/react-native-config/ios/ReactNativeConfig/BuildDotenvConfig.ruby" "${SRCROOT}/.." "${SRCROOT}/tmp.xcconfig"
```

Comment:
it's actually `BuildDotenvConfig.rb`, at least on my machine

Reply:
Yes it's also ` BuildDotenvConfig.rb` on my machine

Reply:
![image](https://user-images.githubusercontent.com/670701/62923187-8a346100-bdf0-11e9-91bd-59772dc35236.png)

It's not on my machine, version `0.11.7`

Reply:
I just created a new app and installed react-native-config and my extension is .ruby. It probably makes sense to have .rb as the extension in the command and include a note that it might be .ruby.